zou/app/blueprints/crud/person.py

```diff
@@ -133,15 +148,20 @@ def post_update(self, instance_dict):
         instance_dict["departments"] = [
             str(department.id) for department in self.instance.departments
         ]
+        if "expiration_date" in data:
+            instance_dict[
+                "access_token"
+            ] = persons_service.create_access_token_from_instance(
```

Comment:
This function naming is weird. Can you explain what it does?

Reply:
It creates the access token for the bot from its instance